src/libraries/System.Runtime.Serialization.BinaryFormat/src/System.Runtime.Serialization.BinaryFormat.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <EnableDefaultItems>true</EnableDefaultItems>
+    <EnableAOTAnalyzer>false</EnableAOTAnalyzer>
```

Comment:
Why does this have AOT analyzer disabled? The payload reader should be 100% trim and AOT compatible.